### DISCUSSION TITLE
Does morphia has live cursor ? Morphia 2.4.x

### DISCUSSION THREAD

**Discussion by SOCRATESZZ** [MorphiaOrg/morphia#3151] [General]
I'd want to know whether Morphia has any capability that enables live cursors. 

- If collections are updated. the data inside cursor should also gets refreshed.
- Similar to tailable cursor in mongodb driver.

**Comment by evanchooly** [comment]
Morphia supports [tailable cursors](https://morphia.dev/morphia/2.4/queries.html#_tailable_cursors), yes.
